ts/call-action-form.tsx

```diff
@@ -104,9 +104,11 @@ export class CallActionForm extends React.Component<Props, State> {
 			} as State);
 			this.props.setModal("loading");
 			var campaign = this.getCampaign(this.props.campaignId);
+			var zipField = (document.getElementsByName('input_zip').item(0) as HTMLInputElement);
 			var data = {
 				"campaignId": campaign.id,
-				"userPhone": phone
+				"userPhone": phone,
+				"zipcode": zipField ? zipField.value : ''
```

Comment:
I didn't want to re-architect the forms while the react code is still a moving target. I thought a one-way data pull from the input wouldn't be a concern, although if you're going full on redux, it might be. 

An empty string is seen by call-congress as no entry on the zip, and it will prompt the user to enter via keypad.